### PR TITLE
Added Windows, and fixed broken links and Languages problem

### PR DESCRIPTION
**Pull request by HBIDamian**
As requested, I have done the Winows part. 

CHANGELOG:
- Added Windows stuff ( but may need corrections )
- Fixed the Language symbol problem
- Fixed some broken links

You may want to check if they is any typos or anything xD

### CODE REVIEW

File: source/installation.rst [hunk 1]
```diff
@@ -6,13 +6,49 @@ Installation
 Installing on Windows
 ---------------------
 
-Download the latest PHP binary from `Bintray <Win-Bintray_>`_.
-Use the installer to install PocketMine-MP.
-The installer may have an outdated version of PocketMine-MP.
-You can download the latest .phar from `Bintray <Phar-Bintray_>`_.
+There are 2 Un-Official PocketMine-MP Installers by TheDeibo Click either x64, or x86 to download your chosen one.
```
Comment (inxomnyaa) [inline source/installation.rst:9]: maybe a new line after your name

Comment (CelticTrinculo) [inline source/installation.rst:9]: sure :) that would be fixed soon. Can you find any typos (etc) in the thing for me? ( I cant see any, myself, but they may be some!

File: source/installation.rst [hunk 2]
```diff
@@ -51,13 +87,13 @@ Install `PocketMine-MP for Android`_ from the Google play.
 Installing manually
 -------------------
 
-Did the installer fail? It is not your taste? YOLO? DIY!
+Did the installer fail? It is not your taste? You cah do it your self!
```
Comment (inxomnyaa) [inline source/installation.rst:90]: can

File: source/installation.rst [hunk 3]
```diff
@@ -6,13 +6,49 @@ Installation
...
-The installer may have an outdated version of PocketMine-MP.
-You can download the latest .phar from `Bintray <Phar-Bintray_>`_.
+There are 2 Un-Official PocketMine-MP Installers by TheDeibo Click either x64, or x86 to download your chosen one.
+
+- One for `x64 <https://github.com/thedeibo/Windows-PocketMine-MP/raw/master/PocketMine-MP-x64.exe>`_
+- One for `x86 <https://github.com/thedeibo/Windows-PocketMine-MP/raw/master/PocketMine-MP-x86.exe>`_
+You can also download the latest .phar from `Bintray <https://bintray.com/pocketmine/PocketMine/PocketMine-MP-phar/view#files>`_.
+These installers provide you with the files you need to make a server!
+
+This installer provides you with these folders and files:
```
Comment (inxomnyaa) [inline source/installation.rst:16]: sounds strange

File: source/installation.rst [hunk 4]
```diff
@@ -6,13 +6,49 @@ Installation
...
-Download the latest PHP binary from `Bintray <Win-Bintray_>`_.
-Use the installer to install PocketMine-MP.
-The installer may have an outdated version of PocketMine-MP.
-You can download the latest .phar from `Bintray <Phar-Bintray_>`_.
+There are 2 Un-Official PocketMine-MP Installers by TheDeibo Click either x64, or x86 to download your chosen one.
+
+- One for `x64 <https://github.com/thedeibo/Windows-PocketMine-MP/raw/master/PocketMine-MP-x64.exe>`_
+- One for `x86 <https://github.com/thedeibo/Windows-PocketMine-MP/raw/master/PocketMine-MP-x86.exe>`_
+You can also download the latest .phar from `Bintray <https://bintray.com/pocketmine/PocketMine/PocketMine-MP-phar/view#files>`_.
+These installers provide you with the files you need to make a server!
```
Comment (inxomnyaa) [inline source/installation.rst:14]: and twice almost the same sentence
|
V

File: source/installation.rst (hunk 1 above)
Comment (inxomnyaa) [inline source/installation.rst:9]: inofficial and unofficial work.
May remove the " - "

**Comment by inxomnyaa** [comment]
^there you go

**Pull request closed without merging**